vowpalwabbit/bfgs.cc

```diff
@@ -385,27 +403,34 @@ void bfgs_iter_middle(vw& all, bfgs& b, float* mem, double* rho, double* alpha,
     mem[(MEM_XT + origin) % b.mem_stride] = (&(*w))[W_XT];
     (&(*w))[W_GT] = 0;
   }
-  for (int j = lastj; j>0; j--)
-    rho[j] = rho[j - 1];
+  for (int j = lastj; j > 0; j--) rho[j] = rho[j - 1];
 }
 
-void bfgs_iter_middle(vw& all, bfgs& b, float* mem, double* rho, double* alpha, int& lastj, int &origin)
+void bfgs_iter_middle(vw& all, bfgs& b, float* mem, double* rho, double* alpha, int& lastj, int& origin)
 {
   if (all.weights.sparse)
     bfgs_iter_middle(all, b, mem, rho, alpha, lastj, origin, all.weights.sparse_weights);
   else
     bfgs_iter_middle(all, b, mem, rho, alpha, lastj, origin, all.weights.dense_weights);
 }
 
-template<class T>
-double wolfe_eval(vw& all, bfgs& b, float* mem, double loss_sum, double previous_loss_sum, double step_size, double importance_weight_sum, int &origin, double& wolfe1, T& weights)
+template <class T>
+double wolfe_eval(vw& all,
+    bfgs& b,
```

Comment:
The many lines here seem a little bit unfortunate.   Fixable?

Reply:
This function has heaps of parameters, I think the extra lines are okay